external/resources/resources.yaml

```diff
@@ -1553,6 +1553,8 @@ pcm-products:
       type: STRING
     locales.en-US.description:
       type: STRING
+    components.*.options[n].id:
+      type: RESOURCE_ID:pcm-products
```

Comment:
```suggestion
```

Let's drop this for now, and create a distinct bug.

Reply:
I might have misunderstood during our discussion, but won't removing this prevent the new runbook action from working correctly?

Reply:
> I might have misunderstood during our discussion

Yes and that is 100% my fault, I was guiding you towards one solution and then telling you a whole bunch of irrelevant details, and then we switched solutions. We should create a new bug for the above, (and I think you did already). 

The above is needed but we need to support some things with regular expressions, and auto complete for it. You don't need it to submit the JSON, it's just used for auto complete, and a few other things.